src/main/java/de/teamlapen/vampirism/blocks/BloodContainerBlock.java

```diff
@@ -135,6 +140,59 @@ public void onBlockPlacedBy(World worldIn, BlockPos pos, BlockState state, Livin
         }
     }
 
+    public static FluidActionResult tryEmptyContainer(@Nonnull ItemStack container, IFluidHandler fluidDestination, int maxAmount, @Nullable PlayerEntity player, boolean doDrain) {
```

Comment:
How are these methods different from the FluidUtil ones?

Reply:
removed